app/views/candidate_mailer/application_under_consideration.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @candidate_name %>,
+
+# Your application is being considered
+
+Your application is now with your teacher training provider(s).
+
+We’ve asked providers to make a final decision within <%= @reject_by_default_days %> working days.
+
+They’ll be in touch with you sooner if they want to arrange an interview.
+
+You can review your applicaiton by signing in to Apply for teacher training:
+
+[Sign in](<%= candidate_interface_sign_in_url %>)
```

Comment:
Should we show the vanilla URL here, as we do in other emails?

Reply:
Yeah, I think so.

Reply:
I thought that was the convention, but then I looked at `submit_application_email.text.erb` where links are inlined. I suppose in that email they're just about part of sentences. 

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/c2d11a372d3f5a9d6f4994e0a87d265be0f3d90d/app/views/candidate_mailer/submit_application_email.text.erb#L21

I'll change it to whatever you suggest.